IceCreamFlavors/DetailView/DetailRatingsView.swift

```diff
@@ -18,7 +18,8 @@ struct DetailRatingsView: View {
         HStack {
             ForEach(rating, id:\.self){item in
                 Image(systemName: item)
-                    .foregroundColor(.green)
+                    .foregroundColor(.purple)
+                    .blur(radius: 5.0)
```

Comment:
Can you change the foreground color to blue?

Reply:
addressed in commit 34ae9dc

Reply:
Great! Thanks for making the change